package.json

```diff
@@ -412,8 +412,6 @@
   },
   "config": {
     "ghooks": {
-      "pre-commit": "npm run lint",
-      "commit-msg": "validate-commit-msg"
```

Comment:
Please put these back. You can run without validation if you pass `-n` to `git commit`